test/metabase/util/schema_test.clj

```diff
@@ -12,7 +13,23 @@
     (is (= (str "value may be nil, or if non-nil, value must satisfy one of the following requirements: "
                 "1) value must be a boolean. "
                 "2) value must be a valid boolean string ('true' or 'false').")
-           (str (su/api-error-message (s/maybe (s/cond-pre s/Bool su/BooleanString))))))))
+           (str (su/api-error-message (s/maybe (s/cond-pre s/Bool su/BooleanString))))))
+    (is (= (str/join "\n"
+                     ["value must be a map with schema: ("
+                      "  a : value must be a map with schema: ("
+                      "    b : value must be a map with schema: ("
+                      "      c : value must be a map with schema: ("
+                      "        d : value must be a map with schema: ("
+                      "          optional-key (optional) : value must be an integer."
+                      "          key : value may be nil, or if non-nil, value must be a boolean."
+                      "        )"
+                      "      )"
+                      "    )"
+                      "  )"
+                      ")"])
+           (str (su/api-error-message
+                 {:a {:b {:c {:d {:key                           (s/maybe s/Bool)
+                                  (s/optional-key :optional-key) s/Int}}}}}))))))
```

Comment:
This test is a lot easier to read than all the other ones. Any reason not to have the other tests like this too?

Reply:
I'll switch over the tests with newlines in that ns to use this style